packages/admin/admin-rte/package.json

```diff
@@ -25,6 +25,7 @@
         "start:types": "tsc --project ./tsconfig.json --emitDeclarationOnly --watch --preserveWatchOutput"
     },
     "dependencies": {
+        "@comet/admin-icons": "workspace:^4.5.0",
```

Comment:
Should this be a dependency or peer dependency?

Reply:
Dependency, otherwise it would be a breaking change, see #1354. I also think that peer dependencies should only be used for packages that don't work if there are multiple versions, for instance, React. I'd rely on PNPM's deduplication for this.